lib/presentation/dashboard_screen/widget/side_menu.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:flutter/material.dart';
+import 'package:sufi_ishq/core/app_export.dart';
+import 'package:sufi_ishq/core/utils/constant.dart';
+import 'package:sufi_ishq/presentation/dashboard_screen/controller/dashboard_controller.dart';
+import 'package:sufi_ishq/presentation/dashboard_screen/widget/side_menu_item.dart';
+
+import '../../../theme/color_initializer.dart';
```

Comment:
avoid relative import and use package import instead

Reply:
done